src/components/common/EffectiveDateTime.vue

```diff
@@ -218,7 +218,7 @@ export default class EffectiveDateTime extends Mixins(DateMixin) {
   }
 
   /** Called when component is mounted. */
-  mounted (): void {
+  protected mounted (): void {
     /** It was decided not doing it for now */
     // this.parseInitialEffectiveDateTime()
```

Comment:
Worth keeping this line in there?

Reply:
This was changed a while back in #183 . Yes, I'll delete this.

Reply:
I slightly changed my mind... `parseInitialEffectiveDateTime()` is now not called and _could_ be deleted also, but there's a bunch of code there that we might want to re-enable at some point. Instead, I'm commenting out the whole block. Have a look and tell me what you think.

Reply:
I'm going to merge this. I can change it in a future PR if desired.